src/services/config/config.service.ts

```diff
@@ -38,5 +40,3 @@ export default class ConfigService {
     return configs[key];
   }
 }
-
-export const configService = new ConfigService();
```

Comment:
That makes sense to me and that would work yes. Ideally you would only use the dependency injection system through the `@Service` decorator and avoid referring to `Container.get(...)` directly. In a hybrid codebase like this I understand why you would export it like this. If we were to implement this I think this would be an appropriate solution for the short term (basically keep exporting it as a singleton).